web-test-runner.config.mjs

```diff
@@ -0,0 +1,17 @@
+import {importMapsPlugin} from '@web/dev-server-import-maps';
+
+export default {
+	nodeResolve: true,
+	plugins: [
+		importMapsPlugin({
+			inject: {
+				importMap: {
+					imports: {
+						'leaflet': '/src/Leaflet.js',
+						'chai': 'https://cdn.jsdelivr.net/npm/@esm-bundle/chai/esm/chai.js'
```

Comment:
It looks like the Web Test Runner docs are outdated, these days `chai` supports ESM fully, and 5.0 is even 'ESM only'. It also looks like `@esm-bundle/chai` is abandoned, so we just install `chai` directly from NPM instead.

Reply:
Fixed in 854624d154cc13d64b9ede7e2c300d4acb6894da